code/_helpers/turfs.dm

```diff
@@ -179,4 +179,27 @@
 
 
 
-	return FALSE
\ No newline at end of file
+	return FALSE
+
+
+
+/proc/isOnShipLevel(var/atom/A)
+	if (A && istype(A))
+		if (A.z in maps_data.station_levels)
+			return TRUE
+	return FALSE
+
+
+//This is used when you want to check a turf which is a Z transition. For example, an openspace or stairs
+//If this turf conencts to another in that manner, it will return the destination. If not, it will return the input
+/proc/get_connecting_turf(var/turf/T)
```

Comment:
Are z transitions from turf-> open space above not a thing?

Reply:
You don't magically move up when you walk under an openspace, so no.

This function is designed to get what the turf sends normal atoms to when they enter it.

I should probably make it account for wormholes too actually